Modules/IO/XML/test/itkFancyStringTest.cxx

```diff
@@ -443,3 +454,65 @@ testFancyStringForStringOperations()
 
   // all testings were successful if reached here
 }
+
+int
+testFancyStringUnequalOperator()
+{
+  {
+    itk::FancyString  s1{ "Hello World!" };
+    const std::string s2{ "Hello World" };
+
+    ITK_TEST_EXPECT_TRUE(s1 != s2);
+
+    const std::string s3{ "Hello world!" };
+
+    ITK_TEST_EXPECT_TRUE(s1 != s3);
```

Comment:
How about adding `ITK_TEST_EXPECT_FALSE(s1 != "Hello World!");`

Reply:
Done in 40b091c.